lib/fathead/webpack_config/output.txt

```diff
@@ -0,0 +1,113 @@
+devServer.watchOptions	A										<section class="prog__container"><p><code>object</code></p><p>Control options related to watching the files.</p><p>webpack uses the file system to get notified of file changes. In some cases this does not work. For example, when using Network File System (NFS). <a href="https://www.vagrantup.com/">Vagrant</a> also has a lot of problems with this. In these cases, use polling:</p><pre><code class="lang-js">watchOptions<span class="token punctuation">:</span> <span class="token punctuation">{</span>\n  poll<span class="token punctuation">:</span> <span class="token boolean">true</span>\n<span class="token punctuation">}</span>\n</code></pre><p>If this is too heavy on the file system, you can change this to an integer to set the interval in milliseconds.</p><p>See <a href="https://webpack.js.org/configuration/watch">WatchOptions</a> for more options.</p></section>	https://webpack.js.org/configuration/dev-server#devserver-watchoptions-
```

Comment:
So I mean `<a href="https://www.vagrantup.com/">Vagrant</a>` should become just `Vagrant`. There shouldn't be links in the description AFAIK